api-tests/legacy-table-creation.js

```diff
@@ -1,7 +1,8 @@
 const { knexMainDB } = require('../server/infra/database/knex');
 
-const createLegacyTable = knexMainDB.schema.raw(
-  `CREATE TABLE IF NOT EXISTS trees
+before(async () => {
```

Comment:
Brought the `before` global hook to this file so that it is run once before all the api tests